azure-pipelines.yml

```diff
@@ -97,17 +96,17 @@ jobs:
     vmImage: 'vs2017-win2016'
   steps:
   - task: CmdLine@2
```

Comment:
Change this to use the `DotNetTool` step instead of command-line?

Reply:
Can't find the docs on `DotNetTool`. BTW Nuke script name is `nuke`, not `dotnet-nuke`, so `dotnet nuke` doesn't work.